crates/nu-command/src/filters/all.rs

```diff
@@ -66,31 +78,79 @@ impl Command for All {
         let ctrlc = engine_state.ctrlc.clone();
         let engine_state = engine_state.clone();
 
-        for value in input.into_interruptible_iter(ctrlc) {
-            if let Some(var_id) = var_id {
-                stack.add_var(var_id, value);
-            }
+        if is_parallel {
+            let par_bridge = input.into_interruptible_iter(ctrlc).par_bridge();
 
-            let eval = eval_block(
-                &engine_state,
-                &mut stack,
-                block,
-                PipelineData::new(span),
-                call.redirect_stdout,
-                call.redirect_stderr,
-            );
-            match eval {
-                Err(e) => {
-                    return Err(e);
+            let result = par_bridge
+                .map(|value| {
+                    let mut stack = stack.clone();
+                    if let Some(var_id) = var_id {
+                        stack.add_var(var_id, value);
+                    }
+
+                    match eval_block(
+                        &engine_state,
+                        &mut stack,
+                        block,
+                        PipelineData::new(span),
+                        call.redirect_stdout,
+                        call.redirect_stderr,
+                    ) {
+                        Err(error) => Value::Error { error },
+                        Ok(pipeline_data) => {
+                            if !pipeline_data.into_value(span).is_true() {
```

Comment:
to me it seems something like: `if !false {return true} else {return false}`...It's not really readable

Maybe we can just do something like this?
```rust
Ok(pipeline_data) => {
    Value::Bool { val: pipeline_data.into_value(span).is_true(), span }
}
```

Reply:
Thanks for the catching!